source/includes/_rars_top_referrers.md

```diff
@@ -0,0 +1,159 @@
+### **Top Referrers**
+<a name="top_referrers"></a>
+
+Describes the distribution of referrers for an advertiser for a given time period as well as attributing the source
+of the referral.  This API endpoint is the data source for the Traffic Detail Report report in client center.
+
+### Resource Overview&nbsp;
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/top_referrer/[gmaid]?[query_params] |
+
+### Parameters&nbsp;
+
+When using the GET method, the results can be filtered using these parameters:
+
+| Param     | Required | Function |
+|-----------|-----|---|
+| start_date | Yes | Restricts the results to those occurring on or after this date and time. |
+| end_date | Yes | Restricts the results to those occurring on or before this date and time. |
+| gmaid | Yes | Restrict results to the specified advertiser |
+|interval_size|No|Return subtotals for the given intervals within the start-end time provided. Accepts calendar_month, calendar_week, or day (Default is day)|
+
+### Response Data Details&nbsp;
+
+
+```shell
+curl --location --request GET 'https://api-stage.gcion.com/apgb2b-reporting/client_reports/top_referrer/TEST_1?start_date=2022-11-10&end_date=2022-11-15&interval_size=calendar_week' \
+--header 'Authorization: xxx' \
+--header 'x-api-key: xxx'
+```
+
+> Example Response
+
+```json
+{
+    "api_name": "top_referrer",
+    "api_run_date": "2022-11-17",
+    "start_date": "2022-11-10",
+    "end_date": "2022-11-15",
+    "time_zone": "America/Los_Angeles",
+    "interval_size": "calendar_week",
+    "global_master_advertiser_id": "TEST_1",
+    "report_data": {
+        "sources": [
+            {
+                "name": "DIRECT",
+                "master_campaign_id": 1,
```

Comment:
By definition DIRECT traffic won't have a MCID associated. Will this be null in that case (apparently that's how it was with the old API), or missing?